test/provider/map_notifier_test.dart

```diff
@@ -21,22 +22,30 @@ void main() {
       mp.setSelectedFloor(8);
       mp.setEnterBuildingVisibility(true);
       mp.setCampusLatLng(LatLng(45.458279, -73.640436));
+      mp.setPopupInfoVisibility(true);
+      mp.setPlaceLatLng(LatLng(45.458279, -73.640436));
 
       expect(mp.showFloorPlan, true);
       expect(mp.selectedFloorPlan, 8);
       expect(mp.showEnterBuilding, true);
       expect(mp.currentCampus, 'LOY');
+      expect(mp.showInfo, true);
+      expect(mp.selectedLatlng, LatLng(45.458279, -73.640436));
+
+      mp.setCampusLatLng(LatLng(45.49558, -73.57801));
+      expect(mp.currentCampus, 'SGW');
 
       mp.setCampusString("SGW");
       expect(mp.currentCampus, 'SGW');
     });
 
     //TODO: Test future methods in map_notifier.dart class
```

Comment:
Is this TODO still applicable, or can it be cleaned up?

Reply:
No it needs to be removed, will not be testing the future methods